manifests/init.pp

```diff
@@ -21,7 +128,7 @@
   $config_dir_group    = 'root',
   $config_dir_mode     = '0755',
   $config_ensure       = 'file',
-  $config_path         = '/etc/eyaml/config.yaml',
+  $config_path         = "${::settings::confdir}/config.yaml",
```

Comment:
`settings` is a class.. this line should stay as it is, as you can already override with Hiera or a parameterized class